ovs/lib/scheduledtask.py

```diff
@@ -247,8 +252,9 @@ def gather_scrub_work():
         local_storage_router = None
         local_scrub_location = None
         local_vdisks_to_scrub = []
-        result_set = ResultSet([])
+        result_set = {}
```

Comment:
Can't we merge these 3 items (```result_set, storage_router_list, scrub_map```
They all contain info about storagerouters
Make it into 1 list with each element being a dict or another list